### ISSUE TITLE
[v2] Port name easily breaks 15 character limit

### ISSUE THREAD

**Issue by dsyer** [telepresenceio/telepresence#1660]
```
telepresence: error: Intercept: Failed to establish intercept: Deployment.apps "configserver" is invalid: spec.template.spec.containers[0].ports[0].name: Invalid value: "tel2mv-configserver": must be no more than 15 characters
```

Thomas Hallgren  on Slack 1 hour ago

> That looks like a bug. We steal 7 characters of the 15 possible, leaving you with only 8 characters for a port name. “configserver” is 12 characters. We shouldn’t do it like that.

**Comment by gtappetize** [comment]
I couldn't find a place to vote/+1 this issue so I'm leaving this comment. We have run into this issue on multiple services. Implementing a different strategy for renaming the port names (maybe rename then trim(15)) would save us a lot of work.

**Comment by khussey** [comment]
This is fixed in 2.1.5, which is now available.
